src/storybook/functions/create-component-story.js

```diff
@@ -26,7 +26,14 @@ export function createStoryMetaSettings({ component, enumPropNamesArray, iconPro
 
   // set enum allowed values inside argsTypes object
   enumPropNamesArray?.forEach(propName => {
```

Comment:
consider renaming the arg name to `prop` so it won't confuse (since it can now be an object..)
```suggestion
  enumPropNamesArray?.forEach(prop => {
```

Reply:
will be fix:)